Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -161,10 +161,10 @@ mainBlockStmt :
     | variableStmt
     | whileWendStmt
     | withStmt
-    | lineSpecialForm
     | circleSpecialForm
     | scaleSpecialForm
     | pSetSpecialForm
+    | lineSpecialForm
```

Comment:
Not sure why this reordering was done. Can you shed some light?

Reply:
Doesn't affect anything really though

Reply:
That is just me forgetting from where I moved the other special forms when trying to resolve the SLL problem.

I will revert that.